README.md

```diff
@@ -2,7 +2,7 @@
 
 ![Vultr](img/logo.png)
 
-The Vultr Resource Provider lets you manage [Vultr](https://vultr.com/) resources.
+The Vultr Resource Provider lets you manage [Vultr](https://www.vultr.com/) resources.
```

Comment:
Is this not the same? 😅

Reply:
Yes and no. There's a redirect, but there are different HTTP response codes for each url.

```
(base) docs $curl -curl -o /dev/null -s -w "%{http_code}\n" www.vultr.com
301
(base) docs $curl -curl -o /dev/null -s -w "%{http_code}\n" https://vultr.com/
301
(base) docs $curl -curl -o /dev/null -s -w "%{http_code}\n" https://www.vultr.com/
200
```